test/level-test.js

```diff
@@ -26,14 +26,14 @@ describe ("Level", function(){
       assert.equal(level.score, 1)
     })
 
-    xit('should prepare a line', function(){
+    it('should prepare a line', function(){
```

Comment:
we'll probobly want to do what you did here in the test above and the test below so that if we monkey with the texts, the test s will still pass.